src/binary-search-tree.js

```diff
@@ -31,8 +64,18 @@ class BinarySearchTree {
   // You'll need the queue-helper file for this. Or could you roll your own queue
   // again. Whatever floats your boat.
   breadthFirstForEach(cb) {
-
-  }
+    const current = [node];
```

Comment:
I think `node` is not defined here? Should probably be `const current = [this];`